docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -1,258 +1,224 @@
 ---
 title: "Go (pull mode)"
 menuTitle: "Go (pull mode)"
-description:  "Instrumenting Golang applications for continuous profiling"
+description: "Set up Go profiling in pull mode"
 weight: 10
 ---
 
 # Go (pull mode)
 
-The Grafana Agent is responsible for pulling profiles from applications and delivering them
-to the Pyroscope database.
+In pull mode, the Grafana Agent periodically retrieves profiles from Golang applications, specifically targeting the
+`/debug/pprof/*` endpoints.
 
-### Prerequisites
+## Set up Go profiling in pull mode
```

Comment:
Because this section does list what a user needs to have before using the procedures on this page, consider using Before you begin (which is the standard heading we use in other procedures). 
```suggestion
## Before you begin
```